internal/config/config.go

```diff
@@ -43,14 +43,19 @@ type ListenerCfg struct {
 // PublisherCfg represent configuration for any publisher types.
 type PublisherCfg struct {
 	Type            PublisherType `valid:"required"`
+	MessageKeyFrom  string
```

Comment:
I actually used a `map[string]string` that materializes a mapping postgres table -> kafka message key, so that it allows to configure the kafka message key for each table. I thing it would be better yet to use a map[string][]string, in order to allow the choosing of several fields for the kafka message key in case of a table that uses a composite primary key (like it is usually done for associative tables)